openedx/stanford/cms/djangoapps/contentstore/views/utilities/bulkupdate.py

```diff
@@ -0,0 +1,257 @@
+"""
+Views related to bulk settings change operations on course problems.
+"""
+
+from django.core.exceptions import PermissionDenied
+from django.core.urlresolvers import reverse
+from django.contrib.auth.decorators import login_required
+from django.http import HttpResponseNotFound
+from django.views.decorators.http import require_GET
+
+from edxmako.shortcuts import render_to_response
+from student.auth import has_course_author_access
+from util.json_request import JsonResponse
+from xmodule.modulestore.django import modulestore
+
+from opaque_keys.edx.keys import CourseKey
+
+from .tasks import bulk_update_problem_settings
+
+SHOW_ANSWER_OPTIONS = [
```

Comment:
Import from cms since we are already importing from cms.

Reply:
Ended up building from a `capa_base.py` file in `common`